generator.yaml

```diff
@@ -21,6 +21,12 @@ operations:
     output_wrapper_field_path: HostedZone
   GetHostedZone:
     output_wrapper_field_path: HostedZone
+  CreateTrafficPolicy:
+    output_wrapper_field_path: TrafficPolicy
+  CreateTrafficPolicyVersion:
+    operation_type: Update
+    resource_name: TrafficPolicy
+    output_wrapper_field_path: TrafficPolicy
```

Comment:
Hmm, OK, so you can't actually update a traffic policy, you can only increment a new policy version, kind of like how [IAM Policies work](https://github.com/aws-controllers-k8s/iam-controller/blob/main/pkg/resource/policy/hooks.go)?